test/Npgsql.Tests/LoadBalancerTests.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace Npgsql.Tests;
+
+public class LoadBalancerTests
+{
+    [Test]
+    public void TestLoadBalance()
+        {
+            var connStringBuilder = "host=127.0.0.3;port=5433;database=yugabyte;userid=yugabyte;password=yugsbyte;Load Balance Hosts=true;Timeout=0;Topology Keys=c1.r1.z1";
```

Comment:
Where is the cluster started? Or is it expected to be running already?
And how do verify the distribution of connection/load is correct?